tests/validation/tests/v3_api/common.py

```diff
@@ -440,7 +469,7 @@ def validate_ingress(p_client, cluster, workloads, host, path,
         curl_args = " -L --insecure "
     if len(host) > 0:
         curl_args += " --header 'Host: " + host + "'"
-    nodes = get_schedulable_nodes(cluster)
+    nodes = get_schedulable_nodes(cluster, "linux")
```

Comment:
"linux" need not be passed in this method. It can be derived from "TEST_OS" variable.

Reply:
Since the ingress-controller only can be deployed in the Linux nodes, we can only choose the Linux nodes to verify the ingress.